components/campaigns/AnnoucementHero.js

```diff
@@ -27,16 +27,16 @@ export default function AnnouncementHero({ className = '', small = false, hideVi
     <Container wide as="section" padding='' className='text-center'>
       <div
         className={`bg-gray-50 border border-gray-200 py-6 rounded ${className} ${
-          small ? 'mb-4' : 'mx-3 mt-3 p-3 mb-6'
-        }`}
+          small ? 'mb-4' : 'mb-12'
```

Comment:
@reachaadrika Why did you changed the CSS here? 
You should check for these classNames inside the test file instead of reverting it to previous classes.

Reply:
changed